src/main/kotlin/io/snyk/plugin/ui/settings/ScanTypesPanel.kt

```diff
@@ -218,26 +219,20 @@ class ScanTypesPanel(
             showSnykCodeAlert("A Snyk Token is necessary to check for Snyk Code enablement.")
             return
         }
-
+        val sastCliConfigSettings: CliConfigSettings? = getSnykApiService().getSastSettings();
+        settings.sastOnServerEnabled = sastCliConfigSettings?.sastEnabled
+        settings.localCodeEngineEnabled = sastCliConfigSettings?.localCodeEngine?.enabled
+        settings.localCodeEngineUrl = sastCliConfigSettings?.localCodeEngine?.url
         val snykCodeAvailable = isSnykCodeAvailable(settings.customEndpointUrl)
         showSnykCodeAlert(
             if (snykCodeAvailable) "" else "Snyk Code only works in SAAS mode (i.e. no Custom Endpoint usage)"
```

Comment:
Is this message still correct?

Reply:
Was wondering if we support ST envs. There is a unit test which confirms we do.

I think this `isSnykCodeAvailable` check can be removed altogether. The only purpose I see in this function is to block custom URLs Snyk has no association with. i.e. a user would put `http://google.com` as a custom URL in the plugin settings. If this happens, the plugin will fail the first request to `http://google.com/api/v1/user.me`.